build_tools/benchmarks/generate_benchmark_comment.py

```diff
@@ -0,0 +1,305 @@
+#!/usr/bin/env python3
```

Comment:
This file is renamed from `build_tools/benchmarks/post_benchmarks_as_pr_comment.py` and refactored. It might be easier to review by diffing with `post_benchmarks_as_pr_comment.py`

Reply:
Thanks for doing the rename in a separate commit. That allows me to see the diff by viewing all but that commit: https://github.com/iree-org/iree/pull/11692/files/a6306d1319c1a43fdc841486612c7af98a602a44..1e055a72d271627211169c092eab9012281b4222